packages/menu/menu-group.md

```diff
@@ -76,3 +121,112 @@ An `<sp-menu-group>` can be used to organize `<sp-menu-item>`s in an `<sp-memu>`
     </sp-menu>
 </sp-popover>
 ```
+
+</sp-tab-panel>
+<sp-tab value="selects-multiple">Multiple</sp-tab>
+<sp-tab-panel value="selects-multiple">
+
+<!-- prettier-ignore -->
+```html demo
+<p>
+    Your favorite parks in New York are: <span id="multiple-group-1-value"></span>
+    <br><br>
+    Your favorite parks in San Francisco are: <span id="multiple-group-2-value"></span>
```

Comment:
At the risk of being a little pedantic here, is it possible to get these values comma _and_ space separated?